### DISCUSSION TITLE
"ConversationsPackage" Package did not load correctly (GitHub Copilot Chat) FOR ONE SOLUTION

### DISCUSSION THREAD

**Discussion by JJsBytes** [community/community#114554] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

when starting visual studio i get the message "ConversationsPackage" Package did not load correctly (GitHub Copilot Chat) 
This only happens for one solution. Other solutions open fine and I can use copilot chat with them. But I cannot with this one solution.

I have tried reparing visual stutio. uninstalling and reinstalling the copilot extensions. 
I tried deleting the .vs folder inside the project. - That didn't work
I even tried making a exporting the project as a template, and crating a new project using that template, but that still gave the error.

Please help. I have forgotten how to work without AI. 😣

**Comment by xLightie** [comment]
I have package loading issue as well.
With log dump:

[Conversations Information] Copilot chat version 0.2.755-beta+8eda486b33.RR (0.2.755.36570). VS: VisualStudio.17.Release/17.9.5+34723.18. Session: d848bc77-96d7-4993-972b-b48c82c87bf8. 
[Conversations Critical] Exception during package initialization: StreamJsonRpc.RemoteInvocationException: 激活“Microsoft.VisualStudio.Copilot.EnvironmentScope (0.3)”服务失败。
   在 Microsoft.ServiceHub.Framework.ServiceJsonRpcDescriptor.LocalProxyGeneration.<>c__DisplayClass39_0.<ReturnedTaskHelperAsync>b__0(Task _)
   在 System.Threading.Tasks.Task.Execute()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.VisualStudio.Conversations.ConversationsPackage.<RegisterContextProvidersAsync>d__23.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.VisualStudio.Conversations.ConversationsPackage.<InitializeAsync>d__21.MoveNext()
RPC server exception:
Microsoft.ServiceHub.Framework.ServiceActivationFailedException: 激活“Microsoft.VisualStudio.Copilot.EnvironmentScope (0.3)”服务失败。
 ---> System.IO.FileNotFoundException: 未能加载文件或程序集“Microsoft.VisualStudio.VCProjectEngine, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a”或它的某一个依赖项。系统找不到指定的文件。
      在 Microsoft.VisualStudio.Copilot.EnvironmentScope.AddProject(Object project)
      在 Microsoft.VisualStudio.Copilot.EnvironmentScope.<InitializeAsync>d__15.MoveNext()
   --- 引发异常的上一位置中堆栈跟踪的末尾 ---
      在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      在 Microsoft.VisualStudio.Shell.ServiceBroker.ServiceBrokerOfExportedServices.<Microsoft-ServiceHub-Framework-IServiceBroker-GetProxyAsync>d__13`1.MoveNext()
   --- End of inner exception stack trace ---
      在 Microsoft.VisualStudio.Shell.ServiceBroker.ServiceBrokerOfExportedServices.<Microsoft-ServiceHub-Framework-IServiceBroker-GetProxyAsync>d__13`1.MoveNext()
   --- 引发异常的上一位置中堆栈跟踪的末尾 ---
      在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      在 System.Threading.Tasks.ValueTask`1.get_Result()
      在 Microsoft.VisualStudio.Utilities.ServiceBroker.GlobalBrokeredServiceContainer.View.<GetProxyAsync>d__19`1.MoveNext()
   --- 引发异常的上一位置中堆栈跟踪的末尾 ---
      在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      在 System.Threading.Tasks.ValueTask`1.get_Result()
      在 Microsoft.VisualStudio.PlatformUI.Packages.ServiceBroker.ServiceBrokerMefExport.<GetProxyAsync>d__6`1.MoveNext()
   --- 引发异常的上一位置中堆栈跟踪的末尾 ---
      在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      在 Microsoft.VisualStudio.Copilot.CopilotSessionProvider.<RegisterScopeAsync>d__16.MoveNext()
   --- 引发异常的上一位置中堆栈跟踪的末尾 ---
      在 System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
      在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
      在 Microsoft.VisualStudio.Copilot.CopilotRegistrationManager.<RegisterScopeAsync>d__5.MoveNext()

**Comment by benmcmorran** [comment]
@spebl could you take a look at this?

**Comment by ayim** [comment]
@xLightie @JJsBytes I recommend filing an issue using Visual Studio's built in "Report a Problem" feature so that we can get more diagnostics information to triage and debug.

![image](https://github.com/community/community/assets/10951869/9f03e0df-40ee-428c-baaa-f299aeb9b73b)


--
Aaron Yim
Product Manage for GitHub Copilot in VS
aayim@microsoft.com

**Comment by benmcmorran** [comment, marked as answer]
@xLightie Can you also check if the "Enable inclusion of additional context for C++" box is checked in Tools > Options > GitHub > Copilot, and if so try unchecking it and see if your problem still persists? That will help us narrow down what could be causing this issue.

![image](https://github.com/community/community/assets/6981284/72f15946-cc7b-4983-9f5a-b54ddd85467e)

cc @spebl

**Reply by xLightie** [reply]
It works (not in Copilot page but same option in Copilot Chat). Thank you sir.

**Reply by christinemaulsby** [reply]
That fixed it for me too! Thank you for the SOLUTION!

**Comment by JJsBytes** [comment]
and after the weekend it just started working again. I hadn't changed anything. I had rebooted multiple times before that and that had not worked.

**Reply by ayim** [reply]
Yep - we rolled out a fix for this issue; glad it's working

**Comment by ayim** [comment]
Hi folks,

This issue affected a small # of C++ users on an early version of 17.11 Preview. An update was rolled out early April so I'm going to close this issue.

--
Aaron Yim
Product Manager for GitHub Copilot in Visual Studio
